src/planscape/impacts/tests/test_views.py

```diff
@@ -30,6 +31,68 @@ def test_create_tx_plan_returns_201(self):
             tx_plan.created_by.id,
         )
 
+    def test_list_txplans_in_scenario(self):
```

Comment:
I'm not sure if this test ought to live here or under scenario viewset tests, or possibly in its own class, since it's a mixed situation.